scripts/triggers/systems/dnd5e.mjs

```diff
@@ -0,0 +1,191 @@
+import { TRIGGERS } from '../../triggers.mjs'
+
+export default function init() {
+  DND5ETriggers.init();
+}
+
+export class DND5ETriggers {
+  /**
+   * Initialize the submodule.
+   */
+  static init() {
+    if (game.system.id !== "dnd5e") return;
+
+    // Inject our triggers into existing categories
+    const effectTriggers = TRIGGERS.find(t => t.label === "EFFECTMACRO.EffectTriggers");
+    const combatantTriggers = TRIGGERS.find(t => t.label === "EFFECTMACRO.CombatantTriggers");
+
+    effectTriggers.options.push(
+      "dnd5e.applyDamage",
+      "dnd5e.shortRest",
+      "dnd5e.longRest",
+      "dnd5e.beginConcentrating",
+      "dnd5e.endConcentration",
+      "dnd5e.rollHitDie"
+    );
```

Comment:
I don't see how these are related to Effects. They don't belong in this same grouping. Additionally, the `dnd5e.healActor` and `dnd5e.damageActor` hooks are probably better than the (kind of useless?) `dnd5e.applyDamage` hook, which also triggers on just 1 client.